setup.cfg

```diff
@@ -1,5 +1,76 @@
 [metadata]
-description-file = README.md
+name = textX
+author = Igor R. Dejanovic
+author_email = igor.dejanovic@gmail.com
+license = MIT
+description = Meta-language for DSL implementation inspired by Xtext
+keywords = parser, meta-language, meta-model, language, DSL
+url = https://github.com/textX/textX
+long_description = file: README
+long_description_content_type = text/markdown
+classifiers =
+    Development Status :: 5 - Production/Stable
+    Intended Audience :: Developers
+    Intended Audience :: Information Technology
+    Intended Audience :: Science/Research
+    Topic :: Software Development :: Interpreters
+    Topic :: Software Development :: Compilers
+    Topic :: Software Development :: Libraries :: Python Modules
+    License :: OSI Approved :: MIT License
+    Operating System :: OS Independent
+    Programming Language :: Python :: 2
+    Programming Language :: Python :: 2.7
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.4
+    Programming Language :: Python :: 3.5
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+
+[options]
+packages = textx, textx.scoping, textx.cli
+zip_safe = False
+install_requires =
+    Arpeggio>=1.9.0
+    click==7.0
+include_package_data = True
+package_dir =
+    textx = textx
+setup_requires =
+    wheel
+
+[options.extras_require]
+dev =
+    textX-dev
+    mkdocs
+    mike
+    twine
+test =
+     flake8
+     tox
+     jinja2
+     coverage
+     coveralls
+     pytest
+
+[options.entry_points]
+console_scripts = 
+    textx = textx.cli:textx
+
+textx_commands = 
+    version = textx.cli.version:version
+    list_languages = textx.cli.discover:list_languages
+    list_generators = textx.cli.discover:list_generators
+    generate = textx.cli.generate:generate
+    check = textx.cli.check:check
+
+textx_generators =
```

Comment:
Maybe also update the demo projects in tests/functional/registration/projects

Reply:
Yeah, good point. It would be good to have a consistency across all examples and tests.